### PR TITLE
fix regression of gh-1150 (#1172)

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/code-instrumentation/setters-test.js [hunk 1]
```diff
@@ -622,3 +622,29 @@ test('setter returns a correct value (GH-907)', function () {
     checkReturnedValue('img', 'src', './path');
     checkReturnedValue('a', 'target', '_blank');
 });
+
+test('body of new document implementation should not throw an error after set him innerHTML (GH-1172)', function () {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/code-instrumentation/setters-test.js]: Rename to `should not throw an error on setting the body.innerHtml when document.body equals null`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
FPR

**Comment by miherlosev** [comment]
ping @churkin

File: test/client/fixtures/sandbox/code-instrumentation/setters-test.js [hunk 2]
```diff
@@ -622,3 +622,29 @@ test('setter returns a correct value (GH-907)', function () {
...
+
+test('should not throw an error on setting the body.innerHtml when document.body equals null (GH-1172)', function () {
+    var iframe = document.createElement('iframe');
+
+    iframe.id = 'test' + Date.now();
+
+    document.body.appendChild(iframe);
+
+    iframe.contentDocument.write(
+        '<head>',
```
Comment (churkin) [inline test/client/fixtures/sandbox/code-instrumentation/setters-test.js]: use `processScript` method

Comment (LavrovArtem) [inline test/client/fixtures/sandbox/code-instrumentation/setters-test.js]: The `processScript` function is called into the `iframe.contentDocument.write` function.
I removed setter from the html.

**Comment by LavrovArtem** [comment]
FPR

**Pull request merged**